pkg/logconfig/config.go

```diff
@@ -31,3 +36,19 @@ const (
 	// Webhook is the name of the override key used inside of the logging config for Webhook Controller.
 	Webhook = "webhook"
 )
+
+func ConfigMapName() string {
+	if cm := os.Getenv(ConfigMapNameEnv); cm != "" {
+		return cm
+	}
+
+	panic(fmt.Sprintf(`The environment variable %q is not set
+
+If this is a process running on Kubernetes, then it should be using the downward
+API to initialize this variable via:
+
+  env:
+  - name: CONFIG_LOGGING_NAME
+  value: eventing-config-logging
+`, ConfigMapNameEnv, ConfigMapNameEnv))
```

Comment:
is there one too many ConfigMapNameEnv here?? seems like there's only one %q.

Also, seems like the indent of the example yaml is off.

Reply:
Done